variables.tf

```diff
@@ -10,6 +10,12 @@ variable "asg_health_check_grace_period" {
   description = "Time (in seconds) after instance comes into service before checking health"
 }
 
+variable "lc_key_name" {
```

Comment:
It doesn't hurt to have additional option and giving possibility to add key if we need one